### PR TITLE
SAK-33252 Stopped bullhorns polling when logged out.

### PR DESCRIPTION
**Pull request by adrianfish**

### CODE REVIEW

File: portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/SkinnableCharonPortal.java [hunk 1]
```diff
@@ -534,7 +534,7 @@ public PortalRenderContext includePortal(HttpServletRequest req,
 		if (prefix != null) portalTopUrl = portalTopUrl + prefix + "/";
 
 		rcontext.put("portalTopUrl", portalTopUrl);
-		rcontext.put("loggedIn", Boolean.valueOf(session.getUserId() != null));
+		rcontext.put("loggedIn", StringUtils.isNotBlank(session.getUserId() != null));
```
Comment (ottenhoff) [inline portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/SkinnableCharonPortal.java]: isNotBlank on the getUserId call?  or the != null call?

Comment (adrianfish) [inline portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/SkinnableCharonPortal.java]: Ha, just saw that. I'm fixing it as I type ...

Comment (adrianfish) [inline portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/SkinnableCharonPortal.java]: Sorted. On the getUserId call is the answer.

Comment (adrianfish) [inline portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/SkinnableCharonPortal.java]: Basically, getUserId was never null, just blank.

File: library/src/morpheus-master/js/src/sakai.morpheus.bullhorns.js [hunk 2]
```diff
@@ -265,7 +265,6 @@
 
     if (portal.loggedIn) {
         updateCounts();
-        //portal.bullhornCountIntervalId = setInterval(updateCounts, portal.bullhorns.pollInterval);
-        portal.bullhornCountIntervalId = setInterval(updateCounts, 10000);
+        portal.bullhornCountIntervalId = setInterval(updateCounts, portal.bullhorns.pollInterval);
```
Comment (ottenhoff) [inline library/src/morpheus-master/js/src/sakai.morpheus.bullhorns.js]: So this change is reverting your previous #4436 ?

Comment (adrianfish) [inline library/src/morpheus-master/js/src/sakai.morpheus.bullhorns.js]: Yes, it was for debug and I shouldn't have committed the hardcoded 10000.

Comment (adrianfish) [inline library/src/morpheus-master/js/src/sakai.morpheus.bullhorns.js]: Not the whole commit. Only that line!

**Pull request merged**